nephio-ansible-install/roles/topo/destroy/tasks/main.yaml

```diff
@@ -0,0 +1,7 @@
+## © 2022 Nokia
+## Licensed under the Apache License 2.0
+## SPDX-License-Identifier: Apache-2.0
```

Comment:
Do the above three lines need to be removed?

Reply:
fixed https://github.com/nephio-project/one-summit-22-workshop/pull/106/commits/30345990bf7948269fe4954e827cb8740fb3984d